packages/components/FocusZone/src/FocusZone.types.ts

```diff
@@ -18,32 +18,55 @@ export interface FocusZoneProps {
   defaultTabbableElement?: React.RefObject<React.Component>;
 
   /**
-   ** Defines which arrows to react to
+   ** Defines which arrow keys to react to
    */
   focusZoneDirection?: FocusZoneDirection;
 
   /**
-   ** If set, the FocusZone will not be tabbable and keyboard navigation will be disabled
+   * Disabled the FocusZone, with slightly different behavior on macOS and win32.
+   * On win32, the FocusZone will not be tabbable and keyboard navigation will be disabled.
+   * on macOS, the FocusZone will "pass through" events, and the children will respond as if there was no FocusZone.
    */
   disabled?: boolean;
 
-  /* Circular Navigation prop */
+  /**
+   * Circular Navigation prop
+   * @platform win32
+   */
```

Comment:
Following the naming for forceFocusMacOS, if this prop is win32 only should we name it isCircularNavigationWin32? Same thing below

Reply:
That would be a change to the public API of the control, which we don't want to do. 

Additions are fine, changes/removals to existing props are not.